inference-engine/tests/functional/plugin/cpu/test_utils/cache_utils.hpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#if defined(__x86_64__) || defined(_M_X64)
+#define DNNL_X64 1
+#else
+#define DNNL_X64 0
+#endif
+
+namespace CPUTestUtils {
+
+class CPUCacheInterface {
```

Comment:
It seems that this class does not store any state. What do you think, m.b. it would be better to move this two methods to the CPUTestsBase class?

Reply:
Thanks for your comment, I moved these methods to the CPUTestsBase class